config/config.go

```diff
@@ -281,11 +278,7 @@ func ParseConfigFile(configPath string, terragruntOptions *options.TerragruntOpt
 
 // Parse the Terragrunt config contained in the given string.
 func parseConfigString(configString string, terragruntOptions *options.TerragruntOptions, include *IncludeConfig, configPath string) (*TerragruntConfig, error) {
-	resolvedConfigString, err := ResolveTerragruntConfigString(configString, include, terragruntOptions)
-	if err != nil {
-		return nil, err
-	}
-
+	resolvedConfigString := configString
```

Comment:
What is the motivation for this change? I suspect this will cause issues downstream...

Reply:
I guess it was a leftover while I was trying to apply all changes that were required to implement late interpolations with the minial effect on the existing code. The variable can be definitly removed and `configString`can be used directly inside the function `parseConfigString`

Reply:
This code has been removed in the current version of PR.